secretmanager/pom.xml

```diff
@@ -101,6 +101,11 @@
           </annotationProcessorPaths>
         </configuration>
       </plugin>
+      <plugin>
+  	<groupId>org.jacoco</groupId>
+  	<artifactId>jacoco-maven-plugin</artifactId>
+  	<version>0.8.12</version>
+      </plugin>
```

Comment:
Suggesting to remove this dependency as we do not use test coverage tools in the repo.

Reply:
This is required to add when i am running with the latest version of java, otherwise it is throwing me compilation errors.

Reply:
I would recommend you remove it and add it back if there are issues in the CI build. Maybe the local config mandates this dependency?

Reply:
Noted.